solr/core/src/java/org/apache/solr/handler/configsets/ListConfigsetsResponse.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.handler.configsets;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class ListConfigsetsResponse extends SolrJerseyResponse {
+
+    //TODO We use both JsonProperty annotations here becaues the v2 serialization code expects Jackson, but
+    // ReflectMapWriter (which we use to squash the typed v2 response into a SolrQueryResponse for v1 code) relies on
```

Comment:
Maybe it could use reflection to support both.  ReflectMapWriter is in SolrJ so it can't reference Jackson.  `CreateAliasPayload` and lots of other DTOs are in SolrJ with annotations.

Reply:
Yep, I think that's the way to go.  I think we should be able to create a ReflectMapWriter impl that can use the "real" Jackson annotations, though I'm only starting in on that now.  🤞

Reply:
(To close the loop: this worked out surprisingly well, problem solved.)